aws/resource_aws_cloudfront_distribution_test.go

```diff
@@ -1422,6 +1460,63 @@ resource "aws_cloudfront_distribution" "main" {
 }
 `, acctest.RandInt(), testAccAWSCloudFrontDistributionRetainConfig())
 
+var testAccAWSCloudFrontDistributionOriginGroupsConfig = `
+variable rand_id {
+	default = %d
+}
+# origin bucket
+%s
+# backup bucket
+%s
+resource "aws_cloudfront_distribution" "failover_distribution" {
+	origin {
+		domain_name = "${aws_s3_bucket.s3_bucket_origin.bucket_regional_domain_name}"
+		origin_id = "primaryS3"
+	}
+  origin {
+    domain_name = "${aws_s3_bucket.s3_backup_bucket_origin.bucket_regional_domain_name}"
+    origin_id = "failoverS3"
+  }
+  origin_group {
+    origin_id = "groupS3"
+    failover_criteria {
+      status_codes = [403, 404, 500, 502]
+    }
+    members {
+      ordered_origin_group_member {
+        origin_id = "primaryS3"
+      }
+      ordered_origin_group_member {
+        origin_id = "failoverS3"
+      }
+    }
+  }
+	enabled = true
```

Comment:
Nit: There seems to be a mix of tabs versus spaces in the test configuration. We prefer to standardize on 2-spaces indentation for HCL configurations.